content/influxdb/cloud-dedicated/reference/cli/influxctl/query.md

```diff
@@ -52,10 +52,10 @@ Default: `table`.
 When using the `table` format, by default, timestamps are formatted as RFC3339
 timestamps. Use the `--time-format` flag to specify one of the available time formats:
 
-- `rfc3339`: _(Default)_
+- `rfc3339nano`: _(Default)_
   [RFC3339-formatted timestamp](/influxdb/cloud-dedicated/reference/glossary/#rfc3339-timestamp)--for example:
   `2024-01-01T00:00:00.000000000Z`
-- `unix`: [Unix nanosecond timestamp](/influxdb/cloud-dedicated/reference/glossary/#unix-timestamp)
+- `unixnan`: [Unix nanosecond timestamp](/influxdb/cloud-dedicated/reference/glossary/#unix-timestamp)
```

Comment:
unixnano

Reply:
Did you also want to update the reference to point to rfc3339-nano as well?

Same question for clustered.

Reply:
Corrected and added.